controllers/host/host_controller.go

```diff
@@ -1590,9 +1586,45 @@ func (r *HostReconciler) ReconcileExistingHost(client *gophercloud.ServiceClient
 		SyncIFNameByUuid(profile, current)
 	}
 
-	err = r.ReconcilePlatformNetworks(client, instance, profile, &hostInfo)
+	host_instance, err := r.CloudManager.GetActiveHost(request_namespace, client)
 	if err != nil {
-		return err
+		msg := "failed to get active host"
+		return common.NewUserDataError(msg)
+	}
+
+	system_info, err := r.CloudManager.GetSystemInfo(request_namespace, client)
+	if err != nil {
+		msg := "failed to get active host"
+		return common.NewUserDataError(msg)
+	}
+
+	if instance.Name == host_instance.Name {
+		// Only reconcile platform networks in active controller reconciliation
```

Comment:
Can we modularize this in a utility method?

Reply:
Done, modularized under `IsActiveHost` method in host controller.